pkg/kubelet/kubelet_pods.go

```diff
@@ -1510,6 +1510,14 @@ func (kl *Kubelet) generateAPIPodStatus(pod *v1.Pod, podStatus *kubecontainer.Po
 			s.Conditions = append(s.Conditions, c)
 		}
 	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodDisruptionConditions) {
+		// copy over the pod disruption conditions
```

Comment:
can we expand this comment to note _why_ we need to copy over the conditions?

Reply:
Done